gammapy/visualization/utils.py

```diff
@@ -20,6 +22,63 @@
 }
 
 
+def plot_rgb(map_, axis="energy", ax=None, **kwargs):
+    """
+    Plot RGB image on matplotlib WCS axes.
+
+    This function is based on the `~astropy.visualization.make_lupton_rgb` function, and it
+    assumes that the input map has exactly three z-axis bins. If that were not the case, you
+    can resample the z-axis of your map as shown in the code example below.
+
+    Parameters
+    ----------
+    map_ : `~gammapy.maps.WcsNDMap`
+        WCS map. If energy edges are not provided, the map should contain exactly 3 energy bins.
+    energy_edges : `~astropy.units.Quantity`, float
+        Energy edges (an iterable of length 4) defining exactly 3 energy bins.
+    ax : `~astropy.visualization.wcsaxes.WCSAxes`, optional
+        WCS axis object to plot on.
+    **kwargs : dict
+        Keyword arguments passed to `~astropy.visualization.make_lupton_rgb`.
+
+    Returns
+    -------
+    ax : `~astropy.visualization.wcsaxes.WCSAxes`
+        WCS axis object
+
+    Examples
+    --------
+    >>> from gammapy.visualization.utils import plot_rgb
+    >>> from gammapy.maps import Map, MapAxis
+    >>> import astropy.units as u
+    >>> map_ = Map.read("$GAMMAPY_DATA/cta-1dc-gc/cta-1dc-gc.fits.gz")
+    >>> axis_rgb = MapAxis.from_energy_edges(
+    >>>     [0.1, 0.2, 0.5, 10], unit=u.TeV, name="energy", interp="log"
+    >>> )
+    >>> map_ = map_.resample_axis(axis_rgb)
+    >>> kwargs = {"stretch": 0.5, "Q": 1, "minimum": 0.15}
+    >>> plot_rgb(map_.smooth(0.08*u.deg), axis=axis_rgb.name, **kwargs)
+    """
+    geom = map_.geom
+    if geom.axes[axis].nbin != 3:
+        raise ValueError("Exactly 3 bins are needed to plot an RGB image")
+
+    data = [data_slice / np.nanmax(data_slice.flatten()) for data_slice in map_.data]
+    data = make_lupton_rgb(*data, **kwargs)
```

Comment:
I think it would be good to make the use of the Lupton method optional here, or introduce a `method` keyword. I think a very good approach for RGB plotting is also plotting `norm` maps, i.e. the flux values with respect to a reference spectral model. In this case no special Lupton approach is required...

Reply:
@adonath Indeed it makes sense. I propose to add this `method` keyword in another PR. Is that OK?